_data/navigation.yaml

```diff
@@ -4,3 +4,17 @@ agile:
     url: /agile/
   - name: More 18F guides
     url: https://18f.gsa.gov/guides/
+
+engineering:
+  - name: Home
+    url: /engineering/
+  - name: About this guide
+    url: https://18f.gsa.gov/engineering/about/
```

Comment:
should I point to directory/file instead of the whole URL. Something to think about. Any thoughts?